spec/meal_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative "../lib/meal.rb"
+require_relative "../lib/menu.rb"
```

Comment:
requiring the menu in here might not be necessary, tried a couple of times to use the tests with only the require to the class I am testing, and it seemed to work. I think maybe RSpec knows to look in the lib folder and find what it requires, if you have a bit of time, maybe take it out and let me know if it still works, really curious about this.